src/webgpu/capability_info.ts

```diff
@@ -228,12 +228,14 @@ export const kTextureViewDimensions = keysOf(kTextureViewDimensionInfo);
 /** Per-GPUVertexFormat info. */
 // Exists just for documentation. Otherwise could be inferred by `makeTable`.
 export type VertexFormatInfo = {
-  /** Number of bytes in each component. */
-  readonly bytesPerComponent: 1 | 2 | 4;
+  /** Number of bytes in each component. It is undefined if format is packed. */
+  readonly bytesPerComponent: 1 | 2 | 4 | undefined;
   /** The data encoding (float, normalized, or integer) for each component. */
-  readonly type: 'float' | 'unorm' | 'snorm' | 'uint' | 'sint';
+  readonly type: 'float' | 'unorm' | 'snorm' | 'uint' | 'sint' | 'packed';
```

Comment:
This type is 'unorm' for each component, isn't it the bytesPerComponent that should be 'packed'?

Reply:
Fixed.